spec/features/tasks/create_spec.rb

```diff
@@ -20,4 +21,10 @@
 
     expect(page).to have_content("Name can't be blank")
   end
+
+  scenario "Completed checkbox is disabeled by default" do
+    visit new_task_path
+
+    expect(page).to have_field('Completed', disabled: true)
```

Comment:
I haven't looked closely, but are there any other ways you could test this Acceptance Criteria? i.e. test this same functionality in a model spec?

Reply:
I am not sure how to do this. 
Could adding validators on the model on create and write these tests for the model and not the disabled checkbox on the form be what you are getting at?

Reply:
Totally! 

I love having the front end validation, but as an honest question, do you think its worth having a feature test over?

I think you should still have the checkbox disabled on the front end, but also throw a model validation & a spec to ensure new records cant be completed.